app/static/js/shared_list.js

```diff
@@ -0,0 +1,36 @@
+var items_num = 1;
+
+document.addEventListener('click', function(e) {
+    if (e.target && e.target.id.includes('btnRemoveItem')){
+        removeItem(e.target.id);
+    }
+    else if (e.target && e.target.id.includes("btnRemoveLabel")) {
+        let parent = document.getElementById(e.target.id).parentElement.id
+        removeItem(parent);
+    }
+    else if (e.target && e.target.id == "btnAddItem"){
+        addItem();
+    }
+});
+
+
+function addItem() {
+    const remove_button = 7;
```

Comment:
Should be REMOVE_BUTTON, no?

Reply:
Umm, I guess your'e right. My bad (again).
I'll change it.